Test fail when marathon url contains a path

I have an endpoint with a marathon URL that looks like 'https://mydomain.tld/marathon/cluster_x'.

The test suite is failing :

```
MARATHON_URL="https://mydomain.tld/marathon/cluster_x" ./test.sh
=== RUN   TestProvider
--- PASS: TestProvider (0.00s)
=== RUN   TestProvider_impl
--- PASS: TestProvider_impl (0.00s)
=== RUN   TestAccMarathonApp_basic
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x525763]

goroutine 21 [running]:
panic(0xaeac80, 0xc820010260)
	/home/xlucas/.gvm/gos/go1.6/src/runtime/panic.go:464 +0x3e6
github.com/Banno/terraform-provider-marathon/vendor/github.com/gambol99/go-marathon.(*APIError).Error(0x0, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/gambol99/go-marathon/error.go:55 +0x123
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform.(*EvalApply).Eval(0xc8202cad80, 0x7fac3858c118, 0xc8202c8270, 0x0, 0x0, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform/eval_apply.go:97 +0xcc3
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform.EvalRaw(0x7fac3854a820, 0xc8202cad80, 0x7fac3858c118, 0xc8202c8270, 0x0, 0x0, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform/eval.go:53 +0x25b
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform.(*EvalSequence).Eval(0xc82028ae80, 0x7fac3858c118, 0xc8202c8270, 0x0, 0x0, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform/eval_sequence.go:10 +0xea
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform.EvalRaw(0x7fac3858c2e0, 0xc82028ae80, 0x7fac3858c118, 0xc8202c8270, 0x0, 0x0, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform/eval.go:53 +0x25b
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform.(*EvalOpFilter).Eval(0xc8202a20c0, 0x7fac3858c118, 0xc8202c8270, 0x0, 0x0, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform/eval_filter_operation.go:37 +0x67
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform.EvalRaw(0x7fac3858c308, 0xc8202a20c0, 0x7fac3858c118, 0xc8202c8270, 0x0, 0x0, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform/eval.go:53 +0x25b
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform.(*EvalSequence).Eval(0xc8202d0c60, 0x7fac3858c118, 0xc8202c8270, 0x0, 0x0, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform/eval_sequence.go:10 +0xea
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform.EvalRaw(0x7fac3858c2e0, 0xc8202d0c60, 0x7fac3858c118, 0xc8202c8270, 0x0, 0x0, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform/eval.go:53 +0x25b
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform.Eval(0x7fac3858c2e0, 0xc8202d0c60, 0x7fac3858c118, 0xc8202c8270, 0x0, 0x0, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform/eval.go:34 +0x6a
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform.(*Graph).walk.func1(0xb97a40, 0xc8202d2810, 0x0, 0x0)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/terraform/graph.go:198 +0x9a9
github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/dag.(*AcyclicGraph).Walk.func3(0xc8202cd260, 0xc8202cab80, 0xc8202cce98, 0xc8202d2930, 0xc8202cd270, 0xb97a40, 0xc8202d2810, 0xc8202d4900, 0xc8202d4960)
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/dag/dag.go:238 +0xeb
created by github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/dag.(*AcyclicGraph).Walk
	/home/xlucas/.gvm/pkgsets/go1.6/global/src/github.com/Banno/terraform-provider-marathon/vendor/github.com/hashicorp/terraform/dag/dag.go:247 +0x715
exit status 2
FAIL	github.com/Banno/terraform-provider-marathon/marathon	0.236s
```
The marathon url gets truncated to 'https://mydomain.tld' and further calls are trying to reach 'https://mydomain.tld/v2/...' instead of 'https://mydomain.tld/marathon/cluster_x/v2/...', ending with a 404 HTTP response server-side and a crash of the provider.

--- a/Godeps/Godeps.json
+++ b/Godeps/Godeps.json
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "github.com/Banno/terraform-provider-marathon",
 	"GoVersion": "go1.6",
-	"GodepVersion": "v62",
+	"GodepVersion": "v74",
 	"Packages": [
 		"./..."
 	],
@@ -130,8 +130,8 @@
 		},
 		{
 			"ImportPath": "github.com/gambol99/go-marathon",
-			"Comment": "v0.0.1-74-ga2b7150",
-			"Rev": "a2b7150eac3175ec91a792acef9dcabd1a9e6e28"
+			"Comment": "v0.3.0-10-ga9a7bc1",
+			"Rev": "a9a7bc12777b95cff48f675bc7173c6e6c530082"
 		},
 		{
 			"ImportPath": "github.com/go-ini/ini",

